Accept AbstractChar in count and findall (::Char, ::String) (#38675)
JuliaLang/julia: NEWS.md

New language features
---------------------

> * count and findall now accept AbstractChar as first argument to search for characters in strings ([#38675](https://github.com/JuliaLang/julia/pull/38675)).

Reviewer: ```suggestion
* `count` and `findall` now accept an `AbstractChar` argument to search for characters in strings ([#38675]).
```
(Note that the issue links are automatically generated, so you shouldn't put the URL in manually.)
